service/src/main/resources/dev-data/reset-database.sql

```diff
@@ -13,21 +11,13 @@ BEGIN
     FROM information_schema.tables
     WHERE table_schema = 'public'
     AND table_type = 'BASE TABLE'
-    AND table_name NOT IN (
-      'flyway_schema_history',
-      'approval_type',
-      'decision_status',
-      'language',
-      'provider_type'
-    )
+    AND table_name <> 'flyway_schema_history'
   );
   FOR sequence IN
     SELECT sequence_name
     FROM information_schema.sequences
     WHERE sequence_schema = 'public'
   LOOP
-    EXECUTE format('ALTER SEQUENCE %I RESTART', sequence);
+    EXECUTE format('SELECT setval(%L, 1, false)', sequence);
```

Comment:
Doesn't this restart all sequences to 1? We've got some sequences which don't start from 1 by default, so this might cause problems

Reply:
Changed the `setval` calls to restart from the correct sequence start value.